README.md

```diff
@@ -12,3 +12,136 @@ libraryDependencies += "com.gu" %% "content-api-firehose-client" % "x.y"
 ```
 
 ## Usage
+
+In order to get started with reading from the content-api firehose you will need to provide us with your AWS account
+number so that we can provide you with permissions. To do so, you may speak to anyone within the content api/off platform
+team and they will be able to help. Or better yet, you can submit the PR yourself! You will need to provide a new 
+parameter to [this] (https://github.com/guardian/crier/blob/master/cloudformation.json) cloudformation file as so:
+
+```
+"TEAM_NAMEAccountNumber": {
+    "Description": "Account number for the TEAM_NAME team",
+    "Type": "String"
+}
+```
+
+And then add a corresponding assumeRole command to the [cross account role] (https://github.com/guardian/crier/blob/master/cloudformation.json#L100)
+as such: 
+
+```
+{
+    "Action": "sts:AssumeRole",
+    "Effect": "Allow",
+    "Principal": {
+        "AWS": {
+            "Fn::Join": [ "", [ "arn:aws:iam::", { "Ref": "TEAM_NAMEAccountNumber" }, ":root" ]]
+        }
+    }
+}
+```
+
+Once this been done, someone on the team will review, merge and update for you to begin. Create an IAM policy in your 
+project's cloudformation file as so: 
+
+ 
+ - STAGE - e.g PROD or CODE
+ - APP_NAME - e.g. my-application
+ - MODE - e.g. live or preview
+ - CAPI_ACCOUNT_NUMBER - The AWS account number for CAPI. Ask someone from the CAPI team to provide you with this.
+ - STREAM_NAME - The AWS Kinesis stream name of the CAPI events stream. Ask someone from the CAPI team to provide you with this.
+
+```json
+  "CrierDynamoDBPolicy": {
+    "Type": "AWS::IAM::Policy",
+    "Properties": {
+      "PolicyName": "CrierDynamo",
+      "PolicyDocument": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "dynamodb:*"
+            ],
+            "Resource": [
+              {
+                "Fn::Join": [
+                  "",
+                  [
+                    "arn:aws:dynamodb:*:YOUR_ACCOUNT_NUMBER:table/content-api-firehose-v2-STAGE_APP_NAME-MODE-",
```

Comment:
Oh, I see. I found this a bit confusing. I thought the STAGE was mistakenly being included twice.